modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+[IMPORTANT]
+====
+Automatic discovery and provisioning is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Use the following procedure to automatically discover local devices, and to automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* You have installed the Local Storage Operator.
+
+* You have attached local disks to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+
+. To display a continuous list of available devices on a node:
+
+.. Log in to the {product-title} web console.
+
+.. Navigate to *Compute* -> *Nodes*.
+
+.. Click the node name that you want to open. The "Node Details" page is displayed.
+
+.. Select the *Disks* tab to display the list of the selected devices.
++
+The device list updates continuously as local disks are added or removed. You can filter the devices by name, status, type, model, capacity, and mode.
++
+[NOTE]
+====
+Results are deleted after they are removed from the node. Symlinks must be manually removed.
+====
```

Comment:
Please remove this note. It does not apply to discovery result output. No symlinks are created for this.

Reply:
Removed

Reply:
Removed